R/notin.R

```diff
@@ -0,0 +1,9 @@
+"%notin%" = function(x, table) {
+  if (is.character(x) && is.character(table)) {
+    result = .Call(Cnotchin, x, table)
+  } else {
+    result = match(x, table, nomatch = 0L) == 0
```

Comment:
0L on the RHS of == too pls

Also I would just drop the `result =` parts and the `return(result)`. The value of `if()` will be returned because that's the last value.